### PR TITLE
samples: tfm for nrf54l15dk

### PR DESCRIPTION
**Pull request by ktaborowski**
move mfg partition to non secure storage in pm static

## CI parameters

```yaml
Github_actions:
  #(branch, hash, pull/XXX/head)
  NRF_revision: v2.8-branch

  # Do not change after creating PR
  Create_NRF_PR: false
Jenkins:
  test-sdk-sidewalk: nrf54l_ns
```

## Description

JIRA ticket: 

## Self review

- [x] There is no commented code.
- [x] There are no TODO/FIXME comments without associated issue ticket.
- [x] Commits are properly organized.
- [x] Change has been tested.
- [x] Tests were updated (if applicable).

### CODE REVIEW

File: samples/sid_end_device/pm_static_nrf54l15dk_nrf54l15_cpuapp_ns.yml [hunk 1]
```diff
@@ -1,43 +1,89 @@
-mfg_storage:
-  address: 0x17a000
-  end_address: 0x17b000
```
Comment (ktaborowski) [inline samples/sid_end_device/pm_static_nrf54l15dk_nrf54l15_cpuapp_ns.yml:3]: mfg_storage same address as in nrf54l15dk (without ns)

File: subsys/config/common/src/app_ble_config.c
Comment (ktaborowski) [inline subsys/config/common/src/app_ble_config.c:1]: NOTE! changes in this file will affect all platforms!

Comment (ktaborowski) [inline subsys/config/common/src/app_ble_config.c:1]: IMPORTANT: changes in this file will affect all platforms!

**Comment by totyz** [comment]
Cert tests: https://jenkins-ncs.nordicsemi.no/job/KRKNG/job/Sidewalk/job/test-sdk-sidewalk/job/test-branch/262/

**Review by totyz — APPROVED** [review]
LQTM.
No regression found in tests.
Smoke tests for _ns platform PASSED.

**Review by RobertGalatNordic — APPROVED** [review]

**Pull request merged**